### PR TITLE
Enable ngspice.

### PR DESCRIPTION
**Pull request by stevefalco**
This requires a (hopefully) temporary patch to suppress "percent
complete" status, which exceeds 1 MB in size and contains many backspace
characters, which display badly.

This patch also turns off the GITHUB plugin, because that currently
won't compile.

### CODE REVIEW

**Comment by stevefalco** [comment]
Seth made a fix for the github plugin compile bug, so I'll run another test build, and push again, when I can confirm that the plugin builds.

**Comment by stevefalco** [comment]
@nickoe and @aimylios, this should be good to go. Please review it and let me know if you want any changes.

I've built it locally, and I can run ngspice without the percent-complete stuff causing trouble.

**Comment by nickoe** [comment]
Looks sorta ok for me. But I think I would like to pull in the fc 27 check as @aimylios did in his patch. 

A minor thing about committing stuff like this. Here you make a change you intend to be permanent (enabling libngspice things), and a change that you intend to remove at a later stage (the patching). This should be two different commits such that it is easy to just revert the temporary commit.

I will attempt to rebase your patches locally.

File: kicad.spec.template [hunk 1]
```diff
@@ -39,12 +41,14 @@ BuildRequires:  libcurl-devel
...
 BuildRequires:  openssl-devel
 BuildRequires:  python2-devel
+BuildRequires:  libngspice-devel
 
 # Documentation
 BuildRequires:  asciidoc
 BuildRequires:  po4a
 
 Requires:       electronics-menu
+Requires:       libngspice
```
Comment (nickoe) [inline kicad.spec.template:51]: It seems that this is not needed according to https://github.com/KiCad/fedora-packaging/pull/24#issuecomment-451116742

**Comment by nickoe** [comment]
I merged this manually and pushed it.

**Comment by nickoe** [comment]
The next offical copr build is expected to include this change.

**Comment by stevefalco** [comment]
I had overlooked the f27 stuff, but it makes sense.

Good point about separating the commits.  I'll follow that practice in the future.

**Comment by nickoe** [comment]
Thank you for your effort.

**Comment by stevefalco** [comment]
A new version of libngspice has been pushed to the fedora 29 testing repository, and I've confirmed that it resolves the problem.  The megabyte of status and backspaces is gone.

The library needs another +1 of karma before it can move to the stable repo, or it will just wait a week, at which point it will move without additional karma.  Either way, once that happens, I'll create a new patch to remove my workaround.

Here is a link to the karma page:

https://bodhi.fedoraproject.org/updates/FEDORA-2019-b71d912484

Also, a KiCad build with my workaround will be eligible to move to fedora 29 stable tomorrow.

**Pull request closed without merging**